### DISCUSSION TITLE
Unit insert selection screen

### DISCUSSION THREAD

**Discussion by tmfset** [odevices/er-301#69] [General]
This would be a nice thing to touch up in 0.7 I think. Having a lot of custom units installed can make the insert screen kind of dense.

Some ideas:
* Filter by author
* Filter by package
* Main screen column view?
* On-screen docs / unit descriptions?

Would be happy to make contributions in this area / do some prototyping.

**Comment by odevices** [comment]
Since this is not directly sampling-related, I would like to target it for [v0.8](https://github.com/users/odevices/projects/2).  No reason not to start thinking about it though!

**Comment by teddarcuri** [comment]
I would also love some more filtering options for the unit insert page.
I am unaware if this option exists today, but I'd love to be able to have a "favorites" category or some sort of tagging mechanism.
